src/modules/mentornity/src/view-wrappers/mentornity-member-home-widgets-wrapper/i18n.ts

```diff
@@ -0,0 +1,56 @@
+export const en = {
+  meetings: 'meetings',
+  meeting: 'meeting',
+  mentees: 'Mentees',
+  mentors: 'Mentors',
+  groups: 'Groups',
+  myMentors: 'My Mentors',
+  myMentees: 'My Mentees',
+  myGroups: 'My Groups',
+  scheduleMeeting: '+Schedule a meeting',
+  seeAll: 'See All',
+  waitingForResponse: ' waiting your response',
+  notMatchedMessage: 'We couldn\'t see a {userType} matched with you at the moment.',
```

Comment:
Bu mesajları olabildiğince menti ve mentor kelimeleri kullanmadan oluşturmak lazım. Eski arayüzün aksine yeni arayüzlerde olabildiğince rollerle ilgili kelime kullanmamaya çalışıyoruz.

Örnek olması için `src/modules/mentornity/src/view-wrappers/mentornity-pair-selection-wrapper/i18n.ts` dosyasına bakabilirisn.

```suggestion
  notMatchedMessage: 'No one is paired with you at the moment.',
```